apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/BigtableClientTest.cs

```diff
@@ -28,6 +28,113 @@ public class BigtableClientTest
         private class TestBigtableClient : BigtableClient { }
 
         [Fact]
+        public async Task CheckAndMutateRow_Validate_TableName()
```

Comment:
Any unit tests for valid calls? Or is it sufficiently trivial to just delegate that to the integration tests?

Reply:
Might be worthwhile just as a sanity check.